curriculum/challenges/english/01-responsive-web-design/css-picasso-painting/part-010.md

```diff
@@ -0,0 +1,59 @@
+---
+id: 60b69a66b6ddb80858c51580
+title: Part 10
+challengeType: 0
+dashedName: part-10
+---
+
+# --description--
+
+The `z-index` property is used to create "layers" for your HTML elements. If you are familiar with image editing tools, you may have worked with layers before. This is a similar concept.
```

Comment:
**Note**: Could only teach the `z-index` topic, once there are enough elements to allow for visual change - do not teach a _visual change_ topic without showing it.

Reply:
I could update this one to "set the z index you'll learn more later" and move the actual instruction content to a place where z-index is relevant.

Reply:
I probably would not set it until it becomes relevant.